TLM/TLM/UI/SubTools/LaneArrowTool.cs

```diff
@@ -132,27 +185,21 @@ public LaneArrowTool(TrafficManagerTool mainTool)
                 if (((netManager.m_segments.m_buffer[HoveredSegmentId].m_startNode == HoveredNodeId)
                      || (netManager.m_segments.m_buffer[HoveredSegmentId].m_endNode == HoveredNodeId))
                     && ((nodeFlags & NetNode.Flags.Junction) != NetNode.Flags.None))
-                {
-                    NetTool.RenderOverlay(
-                        cameraInfo,
-                        ref Singleton<NetManager>.instance.m_segments.m_buffer[HoveredSegmentId],
-                        MainTool.GetToolColor(altDown, false),
-                        MainTool.GetToolColor(altDown, false));
+                    {
```

Comment:
indent here makes no sense, either unindent once, or join with the previous line.  I usually do this part manually, selecting the variant which reads better. The rest of the code is almost always auto-formatted (one block or one function at a time to keep it under control), for simplicity.